app/assess/models/question_field.py

```diff
@@ -16,3 +16,9 @@ def from_json(data: dict):
             field_type=data.get("field_type"),
             answer=data.get("answer"),
         )
+
+    @staticmethod
+    def get_question_fields(json):
+        for pages in json["questions"]:
+            for page in pages["fields"]:
+                return page
```

Comment:
I don't think you'd really need this method because a QuestionField() instance already has title and answer properties so you can just use those like question.title, question.answer

Reply:
okay, so I used the from_json function instead 😄